kinto_http/__init__.py

```diff
@@ -359,6 +395,11 @@ def create_collection(self, collection=None, bucket=None,
         endpoint = self.get_endpoint('collection',
                                      bucket=bucket,
                                      collection=collection)
+
+        logger.info(
+          "Create collection %r for bucket %r with data %r and permissions %r"
```

Comment:
ditto for data and permissions.

Also, I realize that in the Kinto Admin UI we show something like that:  `Create collection blog/articles`. It makes it shorter to read than `%r in bucket %r`. What do you think?